Formula/apache-httpd.rb

```diff
@@ -0,0 +1,128 @@
+class ApacheHttpd < Formula
+  desc "Apache HTTP server"
+  homepage "https://httpd.apache.org/"
+  url "https://www.apache.org/dyn/closer.cgi?path=httpd/httpd-2.4.27.tar.bz2"
+  sha256 "71fcc128238a690515bd8174d5330a5309161ef314a326ae45c7c15ed139c13a"
+
+  bottle do
+    sha256 "d35e67ae745053cda23273a520ead6bde506ec8082057edcd3185c9e36eae483" => :sierra
+    sha256 "4ea7056d6c08f9cf49507869aa8c78e3522c601d87b3347d31561ba05b33d41c" => :el_capitan
+    sha256 "e6ae5c4d38b40bcf3d85877ac117a1ea048de774edec3c2b0ff714bc01a6d40a" => :yosemite
+  end
+
+  deprecated_option "with-http2" => "with-nghttp2"
+
+  depends_on "apr"
+  depends_on "apr-util"
+  depends_on "openssl"
+  depends_on "pcre"
+
+  depends_on "nghttp2" => :optional
```

Comment:
I'd consider adding `depends_on "brotli" => :optional` and adding to the configure options `args << "--with-brotli=#{Formula["brotli"].opt_prefix}" if build.with? "brotli"`, although the configure script will automatically pick it up if it's already installed/linked.

Reply:
Will consider that in a future PR.

Reply:
@kabel What functionality does that provide?

Reply:
It provides the ability to compress content better than zlib, saving wire bits and cpu.

Reply:
Ok, I think that's something we might consider in a future PR.